collections/made-in-algeria/index.md

```diff
@@ -5,8 +5,35 @@ items:
  - Alfanous-team/alfanous
  - GitHubAlgeria/pyIslam
  - mohsenuss91/AlgerianAdministrativeDivision
+ - Alfanous-team/alfanous
+ - idurar/idurar-erp-crm
+ - linuxscout/mishkal
+ - linuxscout/pyarabic
+ - linuxscout/tashaphyne
+ - Hamz-a/frida-android-helper
+ - SofianeHamlaoui/Lockdoor-Framework
+ - assem-ch/django-jet-reboot
+ - assem-ch/arabicstemmer
+ - 01walid/sloughi
+ - OpenDZ/timgad
+ - aissat/easy_localization
+ - Fcmam5/oktob.js
+ - Fcmam5/jelban-js
+ - Fcmam5/nest-problem-details
+ - Fcmam5/hamid.js
+ - Fcmam5/nx-mermaid-grapher
+ - open-minds/awesome-openminds-team
+ - Fcmam5/gharga-cli
+ - Fcmam5/express-response-presenter
+ - Fcmam5/telefonilo.js
+ - Fcmam5/tchamba
+ - linuxscout/alyahmor
+ - linuxscout/yarob
+ - linuxscout/ghalatawi
+ - linuxscout/qalsadi
+ - linuxscout/mishkal
 display_name: Made in Algeria
-created_by: mohammed-elhaouari
+created_by: the-dijkstra
```

Comment:
@the-dijkstra I updated your name here, hope you don't mind :)

Reply:
Thanks mate :pray: